### DISCUSSION TITLE
Activity on forks

### DISCUSSION THREAD

**Discussion by DG12** [community/community#28655] [Codespaces]
Is there a way to sort the forks by activity?

Many people fork a repository and never update it, could they (optionally) be not shown.

**Comment by jfullstackdev** [comment]
if you are at the user's page, you will see that, you can arrange depending on what you want 

![image](https://user-images.githubusercontent.com/47092464/183346413-0e8d0429-13d8-4eda-a9f5-0051bef2cc92.png)

by default, it's arranged by the latest update

**Reply by DG12** [reply]
Thanks for the reply, however, what I am trying to determine is, for a particular repository,  
which users have made made any updates to their fork.
Did they make a pull request  and if they did was it accepted ?

For example on [robert-hh](https://github.com/robert-hh) /[Micropython-Editor](https://github.com/robert-hh/Micropython-Editor)

I see 43 forks, clicking forks/view all  presents a list sorted by user.

When I look at one of the users forks I see the last changes made by robert-hh NOT that user's fork.
Branches show robert-hh's branches.

Clicking on the user
alvor This branch is 11 commits behind rober-hh:master which I think is the commits that Robert-hh has made since the fork was created.

Am I just approaching this the wrong way?

**Reply by jaclynj** [reply]
X commits ahead means commits that are in the original repo that are not in the fork. Commits behind is the # of commits that are in the fork that do not exist on the original repo. Looks like the forks are sorted alphabetically by user. You can click on each fork and look at the latest commit date:
![Screen Shot 2022-08-08 at 5 40 27 PM](https://user-images.githubusercontent.com/5428933/183519596-67c53c78-dfaf-46bc-a10e-0e193c987db8.png)
^ March 2021 in this case

But that will not tell you if they have any changes on a branch or PR.

**Comment by jaclynj** [comment]
Hi! I'm not sure the answer, but you might have better luck if you post in General instead of Codespaces since this question does not have to do with Codespaces.

**Reply by DG12** [reply]
Thank you for your  explanation and suggestion.
